src/api/app/models/branch_package.rb

```diff
@@ -284,12 +284,8 @@ def lookup_incident_pkg(p)
       # only approved maintenance projects
       next unless @maintenance_projects.include? mp.maintenance_project
 
-      # rubocop:disable Metrics/LineLength
-      path = "/search/package/id?match=(linkinfo/@package=\"#{CGI.escape(p[:package].name)}\"+and+linkinfo/@project=\"#{CGI.escape(p[:link_target_project].name)}\""
-      # rubocop:enable Metrics/LineLength
-      path += "+and+starts-with(@project,\"#{CGI.escape(mp.maintenance_project.name)}%3A\"))"
-      answer = Backend::Connection.post path
-      data = REXML::Document.new(answer.body)
+      answer = Backend::Api::Search.incident_packages(p[:link_target_project].name, p[:package].name, mp.maintenance_project.name)
+      data = REXML::Document.new(answer)
```

Comment:
there is still rexml in there? wow

Reply:
Sure! some places still... 

One of the next steps of the Backend::Api refactoring will be using only one lib for all, maybe moving to Nokogiri all the parsing of the backend answers... and moving that parsing inside the lib for providing structured data to the app instead of a string to parse with a random xml lib 👍